main.go

```diff
@@ -130,6 +138,12 @@ func main() {
 		fmt.Fprintf(stdout, "Records to add: %d\n", len(adds))
 		fmt.Fprintf(stdout, "Records to update: %d\n", len(updates))
 		fmt.Fprintf(stdout, "Unchanged records: %d\n", len(records)-len(deleteCandidates))
+
+		if numChanges == 0 && leaveUnknown {
+			fmt.Fprintf(stdout, "\nNote: -leaveunknown option used and this left no changes - exiting.\n")
+			exit(0)
+		}
+
```

Comment:
I don't see how this if-case would ever be reached. `if numChanges == 0 && ... ` will never be true since the outer if-block starts with `if numChanges > 0 && ...`

Reply:
Yea, this doesn't make sense anymore as numChanges now cannot change to 0 after the outer case has let it in (when there are only deltions). I think the hole block should just go...